src/main/java/edu/wright/airviewer2/AIRViewerModel.java

```diff
@@ -40,6 +44,7 @@ int numPages() {
     Image getImage(int pageNumber) {
         BufferedImage pageImage;
         try {
+        	System.out.println("value count"+pageNumber);
             pageImage = renderer.renderImage(pageNumber);
         } catch (IOException ex) {
             throw new UncheckedIOException("AIRViewer throws IOException", ex);
```

Comment:
it would be helpful if print the exception details in the console inside the exception block to understand the cause from the logs

Reply:
Pushed the changes, Please review it